.build/trigger-image-build.sh

```diff
@@ -0,0 +1,4 @@
+curl -H "Authorization: token $GITHUB_TOKEN" \
+    --request POST \
+    --data '{"event_type": "build-latest"}' \
+    https://api.github.com/repos/sakuli/sakuli-docker/dispatches
```

Comment:
Now that I think about it. I don't like this push mechanism. If we follow this pattern, a repo/project has to notify all its dependents actively. This will get out of hands really quick. Is there no possibility to subscribe to something?

Reply:
Okay, I see that there is no such mechanism to subscribe to events to trigger builds. I think it makes no sense to change everything to achieve a decoupled build setup but I'm not happy with this solution either as it might get confusing from where a build was triggered, who triggers who and why etc. When using an event subscription attempt build triggers would be clearly visible with the build config. Just as it is in - I can't believe that im saying this - Atlassian BitBucket.

Reply:
Unfortunately, our current set of services is the limiting factor here. 😞